material-ui/app/src/theme.ts

```diff
@@ -0,0 +1,19 @@
+import { createTheme } from '@mui/material/styles';
+import { red } from '@mui/material/colors';
+
+// Create a theme instance.
+const theme = createTheme({
+  palette: {
+    primary: {
+      main: '#556cd6',
+    },
+    secondary: {
+      main: '#19857b',
+    },
+    error: {
+      main: red.A400,
```

Comment:
Changing `secondary.main` or `error.main` values doesn't seem to affect the example.
Are these needed?